### PR TITLE
[Merged by Bors] - windows CI: use exact same command to prebuild

### PR DESCRIPTION
**Pull request by mockersf**
# Objective

- Running examples on windows crash due to full disk
- The prebuild step was not being reused and consuming extra space

## Solution

- Use the exact same command to prebuild to ensure it will be reused
- Also on linux

### CODE REVIEW

**Review by superdump — APPROVED** [review]
Makes sense to me. :)

**Comment by mockersf** [comment]
bors try

File: .github/workflows/validation-jobs.yml [hunk 1]
```diff
@@ -82,8 +82,9 @@ jobs:
           key: ${{ runner.os }}-windows-run-examples-${{ hashFiles('**/Cargo.toml') }}
 
       - name: Build bevy
+        shell: bash
         run: |
-          cargo build --features "bevy_ci_testing"
+          WGPU_BACKEND=dx12 CI_TESTING_CONFIG=./github/example-run/alien_cake_addict.ron cargo build --example alien_cake_addict --features "bevy_ci_testing"
```
Comment (alice-i-cecile) [inline .github/workflows/validation-jobs.yml:87]: This needs a comment; it's not clear just from reading the code that this *must* be the same as below, or why that's the case.

Comment (mockersf) [reply .github/workflows/validation-jobs.yml:88]: added

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -195,8 +195,9 @@ jobs:
         with:
           toolchain: stable
       - name: Build bevy
+        # this uses the same command as when running the example to ensure build is reused
         run: |
-          cargo build --features "bevy_ci_testing,trace,trace_chrome"
+          WGPU_BACKEND=dx12 CI_TESTING_CONFIG=./github/example-run/alien_cake_addict.ron cargo build --example alien_cake_addict --features "bevy_ci_testing,trace,trace_chrome"
```
Comment (superdump) [inline .github/workflows/ci.yml:200]: dx12 on linux? :)

**Review by superdump — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (mockersf) [reply .github/workflows/ci.yml:200]: I need to stop copy pasting 😢

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -195,8 +195,9 @@ jobs:
         with:
           toolchain: stable
       - name: Build bevy
+        # this uses the same command as when running the example to ensure build is reused
         run: |
-          cargo build --features "bevy_ci_testing,trace,trace_chrome"
+          TRACE_CHROME=trace-alien_cake_addict.json CI_TESTING_CONFIG=./github/example-run/alien_cake_addict.ron cargo build --example alien_cake_addict --features "bevy_ci_testing,trace,trace_chrome"
```
Comment (superdump) [inline .github/workflows/ci.yml:200]: Hmm should this be .github/example-run/…?

Comment (mockersf) [reply .github/workflows/ci.yml:200]: nop

**Review by alice-i-cecile — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 3 above)
Comment (mockersf) [reply .github/workflows/ci.yml:200]: yes

**Review by superdump — APPROVED** [review]

**Comment by superdump** [comment]
🤞

**Pull request closed without merging**